api/metadata_processor.go

```diff
@@ -322,6 +322,51 @@ func doProcess(exec boil.Executor, metadata CITMetadata, original, proxy *models
 			return nil, errors.Wrap(err, "Associate tags")
 		}
 	}
+	if len(metadata.Likutim) > 0 {
+		log.Infof("Associating %d likutim", len(metadata.Likutim))
+		likutim, err := models.ContentUnits(exec,
+			qm.Select("distinct on (\"content_units\".id) \"content_units\".*"),
+			qm.WhereIn("uid in ?", utils.ConvertArgsString(metadata.Likutim)...)).
```

Comment:
Maybe enforce type likutim here ?
Otherwise you might get UID of existing units but not of type likutim.